king_phisher/client/dialogs/campaign_selection.py

```diff
@@ -57,13 +57,14 @@ class CampaignSelectionDialog(gui_utilities.GladeGObject):
 	gobject_ids = (
 		'button_new_campaign',
 		'button_select',
-		'checkbutton_show_expired',
 		'drawingarea_color_key',
 		'label_campaign_info',
-		'treeview_campaigns'
+		'treeview_campaigns',
+		'menubutton_filter'
 	)
+
 	top_gobject = 'dialog'
-	top_level_dependencies = ('StockAddImage',)
+	top_level_dependencies = ('StockAddImage')
```

Comment:
This actually changes the meaning, in Python `('StockAddImage') != ('StockAddImage',)`. The first evaluates to a string, while the second (due to the trailing comma) evaluates to a tuple containing a single string object.